standards-catalogue/nix/sources.nix

```diff
@@ -0,0 +1,174 @@
+# This file has been generated by Niv.
```

Comment:
:question: should this be commited?

Reply:
Yes, this is what Niv uses to pin down the version of nixpkgs which will have stronger guarantees than just plain nix